protocol-units/da-sequencer/node/src/block.rs

```diff
@@ -1,4 +1,9 @@
 use movement_types::block::Block;
+use serde::{Deserialize, Serialize};
+
+use crate::error::DaSequencerError;
+
+const MAX_SEQUENCER_BLOCK_SIZE: u64 = 1_000_000; // 1 MB
```

Comment:
Where does this number come from? Is it related to the Celestia blob size?

Reply:
We should add a `// TODO: use a sensible value for the max sequencer block size`.